### DISCUSSION TITLE
Is DPI limit in MagickImage.write?

### DISCUSSION THREAD

**Discussion by sihj2002** [dlemstra/Magick.NET#618] [General]
I use Magick.net now. But I have a problem with fix Density.

           string filepath = "C:/.../.tif";

            MagickImage img = new MagickImage(filepath);

            img.Density = new Density(720 , 360, DensityUnit.PixelsPerInch);

            img.Depth = 1;

            img.Write(Path.GetDirectoryName(filepath) + "/" + Path.GetFileNameWithoutExtension(filepath) + "test.tif");'

Code goes well but Density change like

`
   img.Denstiy = new Density(1800,360,DensityUnit.PixelsPerInch);
`

I get wrong DPI. Like this.
![magick](https://user-images.githubusercontent.com/64253749/80196786-c9da6a80-8658-11ea-971c-a9d2b3274304.jpg)

I want to fix DPI (1800x360) but I get (1800x1800).

How can I solve this problem?

**Comment by dlemstra** [comment]
What are you using to read that information?

**Reply by sihj2002** [reply]
I read DPI information like below.

![DPI](https://user-images.githubusercontent.com/64253749/80323236-31650580-8865-11ea-96a0-42c13cd17716.jpg)

**Comment by sihj2002** [comment]
When I use magick.net to read DPI information, it works well. There are 3 tif files.

![magick question2](https://user-images.githubusercontent.com/64253749/80445825-2935c480-8950-11ea-85ec-9ba7a0448a3e.jpg)

It is weird when I want see an image information in desktop.
Below picture is showing DPI setting is changed.
I fix dpi for each picture. ( 1440x360 , 1441x360 , 1800x360)
![magick question](https://user-images.githubusercontent.com/64253749/80446034-afeaa180-8950-11ea-914f-a4fc9f26e06e.jpg)


I don't know why vertical resolution is different with I want.

I want to get proper DPI information when I open the properties window with right click on the desktop.


Do you have idea that I can see correct DPI in properties window?

**Reply by dlemstra** [reply]
Maybe this is a bug in the windows file viewer? I suspect that when you make the value to low (difference between horizontal and vertical) it will display it differently from how it is stored in the file. No idea why this is happening though.

**Reply by sihj2002** [reply]
Okay.. thank you for kind answer.
